bindings/java/src/org/sleuthkit/datamodel/DataSource.java

```diff
@@ -74,6 +74,18 @@
 	 * @throws TskCoreException Thrown if the data can not be written
 	 */
 	void setAcquisitionDetails(String details) throws TskCoreException;
+
+	/**
+	 * Sets the acquisition settings used as well as module name and module version.
+	 * 
+	 * @param acquisitionSettings Any settings specific to the acquisition. May
+	 *                            be Null.
+	 * @param moduleName          The module name. May be Null
+	 * @param moduleVersion       The module's version number. May be Null.
```

Comment:
I think we should be more explicit about what module name and module version mean in this documentation. I actually don't know.